### DISCUSSION TITLE
Issue with flask deployment

### DISCUSSION THREAD

**Discussion by Judimax** [vercel/vercel#8348] [Help]
## Expected Behavior

* I should be able to deploy  the flask app to vercel and it should work sucessfully. 

## Current Behavior

I get these errors
![1659959767543](https://user-images.githubusercontent.com/21322737/183413350-7aad7f3f-06ee-4503-b778-48abffc59f1e.png)
![1659959739463](https://user-images.githubusercontent.com/21322737/183413355-2fbd2629-dda7-410b-ba34-a8d9b51c943d.png)

## Possible Solution

* I need guidance to help vercel treat my app not as a serverless function but a wsgi app

## Steps to Reproduce

1. update vercel.json
```json
{
  "version": 2,
  "public":true,
  "builds": [
      {
          "src": "index.py",
          "use": "@vercel/python"
      }
  ],
  "routes": [
      {
          "src": "/(.*)",
          "dest": "index.py"
      }
  ]
}
```
2. npx vercel deploy
3.
4.

## Environment
|property|value|data|
|:------|:------:|------|
|Vercel CLI| v27.3.7||

|local python env|3.10.6||
|vercel python|@vercel/python||
|flask|2.2.1||
||||

**Comment by Judimax** [comment]
Also this is a mimial reproduction. https://vercel.com/michaelodumosu57/vercel-flask-minimal-reproduction/4EksVpZwGkjbXbV9u1QRUEKiRsYT

**Reply by amyegan** [reply]
Thanks for sharing this! It looks like you're already working with someone on the ticket you submitted. We can share the answer here to help the community once you get it resolved.

**Comment by Judimax** [comment]
Are you talking about the issue connected to this ticket or the private support email, either I need to look at the deploy logs to see the specific programming error that is happening For example according to the below issues, I need to see what is going on in my minimal repro vercel figured out that this is a wsgi app, but working with my development toochain, vercel can't get it figured out
![image](https://user-images.githubusercontent.com/21322737/184452473-19cd6325-770c-429a-b84d-d4bb451eb146.png)
![image](https://user-images.githubusercontent.com/21322737/184452539-f3ee2ce9-51c8-434c-a3c1-f3286b81827f.png)
